packages/amplication-client/src/Plugins/PluginsCatalog.tsx

```diff
@@ -30,7 +42,15 @@ const PluginsCatalog: React.FC<Props> = ({ match }: Props) => {
   const handleInstall = useCallback(
     (plugin: Plugin, pluginVersion: PluginVersion) => {
       const { name, pluginId, npm } = plugin;
-      const { version, settings } = pluginVersion;
+      const { version, settings, configurations } = pluginVersion;
+      const requireAuthenticationEntity = configurations
+        ? configurations["requireAuthenticationEntity"]
```

Comment:
What if there is a configurations object (not null) but the configuration does not contain a "requireAuthenticationEntity" key? 
That should be optional too.